rust/src/lib/query_apply/net_state.rs

```diff
@@ -93,7 +94,9 @@ impl NetworkState {
             );
         }
         if !self.kernel_only {
-            self.apply_with_nm_backend()
+            with_retry(RETRY_NM_INTERVAL_MILLISECONDS, RETRY_NM_COUNT, || {
```

Comment:
It looks as though we're jumping directly to a two-second backoff?  That seems likely to be more than necessary, and would contribute to a meaningful boot delay.  Would it make sense to add more retries and reduce the interval between them?

Reply:
Retrying too frequent might lead to more race problems when user are creating 1000 VLANs. The retry here is more focusing on handle *unknown* problems. For this known problem, I have added a 2 seconds delay in `nmstatectl service` to avoid the meaningless retry. We may remove this sleep once NM fixed their problem.